includes/Checker/Checks/Plugin_Repo/Plugin_Readme_Check.php

```diff
@@ -692,6 +692,35 @@ private function check_for_contributors( Check_Result $result, string $readme_fi
 				'',
 				6
 			);
+
+			return;
+		}
+
+		$restricted_contributors = array(
```

Comment:
Could we add a _warning_ for wordpressorg? Sometimes people mistakenly add this user as a contributor, even though it‘s not a feature/community plugin